src/main/java/tracing/SNT.java

```diff
@@ -0,0 +1,26 @@
+package tracing;
+
+import org.scijava.util.VersionUtils;
+
+import ij.IJ;
+
+/** Static utilities for SNT **/
+public class SNT {
+	public static final String VERSION = getVersion();
+
+	private SNT() {
+	}
+
+	private static String getVersion() {
+		return VersionUtils.getVersion(tracing.SimpleNeuriteTracer.class);
+	}
+
+	protected static void error(final String string) {
+		IJ.error("Simple Neurite Tracer v" + getVersion(), string);
+	}
+
+	protected static void log(final String string) {
+		System.out.println("[SNT] " + string);
```

Comment:
OK. Thanks a lot for the input. It was done on e4528a9. I've also started to replace `IJ.log()` calls throughout the several classes of the plugin, and initiated descriminating the type of debug level (error, info, warning), etc. The SNT class now looks [like this](https://github.com/tferr/Simple_Neurite_Tracer/blob/dev/src/main/java/tracing/SNT.java).

Note that there are _lots_ of things that could be moved to scijava right now (namely routines dealing with files and routines handling the coloring of paths) that should adopt [scijava.util](http://javadoc.imagej.net/SciJava/index.html?org/scijava/util/). It would be fairly easy, but IMHO that effort should be initiated once SNT is rewritten for ImageJ2.